client/src/modules/cohort/components/CohortHeader.jsx

```diff
@@ -197,9 +205,9 @@ class CohortHeader extends PureComponent {
                 {description}
               </p>
             )}
-            {!description && (
+            {!description.trim() && (
```

Comment:
is this necessary to trim description value in render? Could we do this before it's stored in state?